CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+## Unreleased
+
+### Features
+
+- Sentry Replay Support ([#354](https://github.com/getsentry/sentry-wizard/pull/354))
+
+How to use:
+
+Replay, profiling and performance monitoring are bundled into Sentry Cordova, all you need to do is to initialise the SDK, include the desired feature and use the sampling filters:
+
+```javascript
+/***
+ * @type {import("sentry-cordova")}
+ */
+  var Sentry = cordova.require("sentry-cordova.Sentry");
+  Sentry.init({
+    dsn: YOUR_DSN,
+    integrations: [
+      // Replay integration.
+      Sentry.replayIntegration({
+        maskAllText: false,
+        blockAllMedia: true,
+      }),
+      // Tracing integration.
+      Sentry.browserTracingIntegration(),
+    ],
+    // Replay sampling filters.
+    replaysSessionSampleRate: 1,
+    replaysOnErrorSampleRate: 1,
+
+    // Tracing sampling filter.
+    tracesSampleRate: 1,
+  });
+```
+
+### Dependencies
+
+- Bump Sentry JavaScript SDK to `7.118.0` ([#354](https://github.com/getsentry/sentry-cordova/pull/354))
```

Comment:
The latest v7 is `7.119.0`, let's bump it to that (if there are no issue with that for cordova).